index.bs

```diff
@@ -841,7 +861,11 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 
 							Issue: It should be possible to read the data asynchronously from the system clipboard after the author calls getType, however, this set of steps implies that data will be provided at the time of read.
 						
-						1. The user agent, MAY sanitize |representation|'s [=representation/data=], unless |representation|'s [=representation/MIME type=]'s essence is "image/png", which should remain unsanitized to preserve meta data.
+						1. The user agent, MAY sanitize |representation|'s [=representation/data=], unless it satisfies the below conditions:
+						
+							1. |representation|'s [=representation/MIME type=]'s [=MIME type/essence=] is "image/png", which should remain unsanitized to preserve meta data.
+							
+							1. If |format| is not empty and |representation|'s [=representation/MIME type=]'s [=MIME type/essence=] is equal to |format|.
```

Comment:
Is this trying to say that the representation's MIME type is amongst the formats inside the sequence of `DOMString`s, which is named `unsanitized`? https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/clipboard/clipboard.idl;l=1?q=clipboard.idl&sq=&ss=chromium

Reply:
basically yes. We extract the first DOMString in the `If |formats| is not empty, then` steps and then try to match the MIME type with it in this highlighted step. In this case it will be just `text/html` mime type.